### PR TITLE
Issue #6818: fix expanded style in AnnotationUseStyleCheck

### PR DESCRIPTION
**Pull request by strkkk**
#6818 also removes usage ob branchContains for #5124

Report - https://strkkk.github.io/5124_anno/

### CODE REVIEW

**Comment by strkkk** [comment]
@romani  @rnveach can you please check what is wrong with wercker build?

**Comment by rnveach** [comment]
Wercker is false error. Page shows everything passed.

**Comment by rnveach** [comment]
@strkkk Since you have 2 commits, lets split into 2 PRs. 1 already shows it will have new violations.
We will need regression for both checks.

**Comment by strkkk** [comment]
@rnveach ok, done #6811

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheckTest.java [hunk 1]
```diff
@@ -151,6 +151,9 @@ public void testStyleExpanded() throws Exception {
             "39: " + getCheckMessage(MSG_KEY_ANNOTATION_INCORRECT_STYLE, "EXPANDED"),
             "41: " + getCheckMessage(MSG_KEY_ANNOTATION_INCORRECT_STYLE, "EXPANDED"),
             "58: " + getCheckMessage(MSG_KEY_ANNOTATION_INCORRECT_STYLE, "EXPANDED"),
+            "63: " + getCheckMessage(MSG_KEY_ANNOTATION_INCORRECT_STYLE, "EXPANDED"),
+            "71: " + getCheckMessage(MSG_KEY_ANNOTATION_INCORRECT_STYLE, "EXPANDED"),
+            "75: " + getCheckMessage(MSG_KEY_ANNOTATION_INCORRECT_STYLE, "EXPANDED"),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheckTest.java]: > `@AnnotationWithAnnotationValue(@Another)`
> `@AnnotationWithAnnotationValue(@Another())`

Please confirm. Are you printing a violation on `AnnotationWithAnnotationValue` or `Another` or both?

Comment (strkkk) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheckTest.java]: @rnveach on both.

**Comment by strkkk** [comment]
@romani done. #6806

**Comment by strkkk** [comment]
> please rebase to resolve Travis

done, ci is green

**Pull request merged**